### PR TITLE
Trim address on active address adding and don't crash on non-valid address

### PR DESCRIPTION
**Pull request by KillariDev**
fix https://github.com/DarkFlorist/TheInterceptor/issues/557

### CODE REVIEW

File: app/ts/utils/bigint.ts [hunk 1]
```diff
@@ -84,6 +84,13 @@ export function checksummedAddress(address: bigint) {
 	return ethers.getAddress(addressString(address))
 }
 
+export function stringToAddress(addressString: string | undefined) {
+	if (addressString === undefined) return undefined
+	const trimmedAddress = addressString.trim()
+	if (!ethers.isAddress(trimmedAddress)) return undefined
```
Comment (MicahZoltu) [inline app/ts/utils/bigint.ts:90]: Feels like this could just be a regex instead.

Comment (KillariDev) [reply app/ts/utils/bigint.ts:90]: I feel regex is more complex than using ethers method?

Comment (MicahZoltu) [reply app/ts/utils/bigint.ts:90]: 🤷 
```ts
/0x[a-fA-F0-9]{40}/.test(trimmedAddress)
ethers.isAddress(trimmedAddress)
```
If Ethers is doing checksumming then I agree with you.

Comment (KillariDev) [reply app/ts/utils/bigint.ts:90]: yeah, I feel `isAddress` is more clear than regexp there. If done with regexp, I would wrap that in "isAddress" function anyway to comment it

**Review by MicahZoltu — APPROVED** [review]
Sorry, not sure why this was a comment instead of an approval.

**Pull request merged**